### DISCUSSION TITLE
Data storage support ETCD, REDIS

### DISCUSSION THREAD

**Discussion by fairyqb** [Kong/kong#7897] [Ideas and feature requests]
Based on the existing infrastructure, reduce management and maintenance costs

**Comment by mayocream** [comment]
Hi @fairyqb, thanks for sharing your ideas.😊

We don't have plans to support other data storage like etcd and redis **yet**.

Currently, the team is working on https://github.com/Kong/kong/pull/8224 to use LMDB as the DB-less backend.
You might want to look at [Kong DB-less and Declarative Configuration](https://docs.konghq.com/gateway/2.7.x/reference/db-less-and-declarative-config/#main).

And if you are using Kong inside your K8s cluster, look at the [KIC DB-less deployments](https://docs.konghq.com/kubernetes-ingress-controller/2.0.x/concepts/deployment/#without-a-database).
